packages/sdk/src/types/AccountAddress.ts

```diff
@@ -24,6 +38,8 @@ class AccountAddress {
  * - The base58 string is a length of 50 (encoding exactly 32 bytes).
  */
 export type Type = AccountAddress;
+export const instanceOf = (value: unknown): value is AccountAddress =>
+    value instanceof AccountAddress;
 
 /**
  * Type guard for AccountAddress
```

Comment:
We probably need to remove this function `isAccountAddress` since it can cast plain objects to be of `AccountAddress` without actually being an object of the class.